lib/player.rb

```diff
@@ -2,15 +2,16 @@
 require './lib/board'
 
 class Player
-  attr_reader :board, :cruiser, :submarine, :ships
+  attr_reader :board, :ships
 
-  def initialize(board_size = 4)
+  def initialize(game_ships = {'Submarine' => 2, 'Cruiser' =>3 }, board_size = 4)
```

Comment:
This, this was solid!!! Had an oh shit moment earlier and you, as usual, had thought of this. Bad.ass.